app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<body>
+    <% @users_all.each do |user| %>
+        <%= link_to "/users/#{user.id}" do %>
+        <%= render "partials/user_container", user: user %>
+        <% end %>
+    <% end %>
+</body>
```

Comment:
- [ ] Kindly check that the right model is being called with the right type, I `.each` will only work on an array type 
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/67957533/233354912-846d5f10-621d-449c-877b-db033a3ae2ef.png">